### PR TITLE
Update client registration documentation

### PR DESCRIPTION
**Pull request by marcelovilaca**
Fixed old Mitre entries and add new Dashboard images

### CODE REVIEW

File: content/en/docs/tasks/user/client-registration/_index.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ weight: 2
 ---
 
 IAM exposes the OpenID Connect/OAuth dynamic client registration functionality
-offered by the [MitreID OpenID Connect][mitreid] server libraries.
+on it's dashboard's main page (it has been migrated by dashboard).
```
Comment (federicaagostini) [inline content/en/docs/tasks/user/client-registration/_index.md]: it's -> its

File: content/en/docs/tasks/user/client-registration/_index.md [hunk 2]
```diff
@@ -17,55 +17,55 @@ A new client can be registered in the IAM in two ways:
...
-![Mitre client reg](../images/mitre-client-reg-5.png)
+![client reg](../images/new-client-reg-4.png)
 
 IAM will then generate client credentials for your client and other information
 that will be displayed as follows:
 
-![Mitre client reg](../images/mitre-client-reg-6.png)
+![client reg](../images/new-client-reg-5.png)
+
+At __Main tab__ one can copy the `Registration Client URI` and paste on brouser to get the client information in JSON format
```
Comment (federicaagostini) [inline content/en/docs/tasks/user/client-registration/_index.md]: brouser -> browser

**Comment by federicaagostini** [comment]
The Registration Access Token is not used anymore by the user to modify the client. 
Then maybe that section can be removed (starting from line 55)?

**Comment by rmiccoli** [comment]
Maybe it would be better to always use Admin/Test User in screenshots so we avoid deleting the user's name?

**Pull request merged**